package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wharf-website",
+  "name": "wharfkit-website",
```

Comment:
Not sure it matters a ton though, since this isn't an npm package... but to match our other naming conventions, I think this should just be: 

`@wharfkit/website`

https://github.com/wharfkit/session/blob/1e36a4ff2e67a8f005888f80f780e959b3a7fc6b/package.json#L2

Reply:
Love it.
```suggestion
  "name": "@wharfkit/website",
```